*: clarify rules and policies (#7823)
pingcap/docs: information-schema/information-schema-placement-rules.md

# PLACEMENT_RULES

> The `PLACEMENT_RULES` table provides information on all explicitly configured [Placement Rules in SQL](/placement-rules-in-sql.md), i.e. all placement policies.

Reviewer: ```suggestion
The `PLACEMENT_RULES` table provides information on all explicitly configured [Placement Rules in SQL](/placement-rules-in-sql.md), which means all placement policies.
```

## Examples

> The `PLACEMENT_RULES` table only shows explicitly configured policies. To see the canonical version of placement rules (including placement policies attached to tables), use the statement `SHOW PLACEMENT` instead:

Reviewer: In https://github.com/pingcap/docs-cn/pull/8230/files#diff-676074174d07d2710042c3cd3be1c902e9c4dd2035ebfca7806e23f8370f671aR42, the Chinese is "已显式配置的规则"
```suggestion
The `PLACEMENT_RULES` table only shows explicitly configured rules. To see the canonical version of placement rules (including placement policies attached to tables), use the statement `SHOW PLACEMENT` instead:
```
